src/unix/linux_like/android/b64/aarch64/mod.rs

```diff
@@ -433,6 +433,10 @@ pub const SYS_syscalls: ::c_long = 451;
 pub const PROT_BTI: ::c_int = 0x10;
 pub const PROT_MTE: ::c_int = 0x20;
 
+// From NDK's asm/auxvec.h
+pub const AT_SYSINFO_EHDR: ::c_ulong = 33;
+pub const AT_VECTOR_SIZE_ARCH: ::c_ulong = 2;
```

Comment:
Is `AT_MINSIGSTKSZ` useful? Since it is also in https://android.googlesource.com/platform/bionic/+/refs/tags/android-14.0.0_r61/libc/kernel/uapi/asm-arm64/asm/auxvec.h

Reply:
This constant is in `src/unix/linux_like/android/mod.rs`

Reply:
Ah didn't catch that it was `ifndef` in `linux/auxvec.h`, thanks